master-test-cases.yml

```diff
@@ -408,13 +408,14 @@ body:
     negative:
         - '{}'
         - '[]'
-  - type: OptionalBinaryAliasExample
-    positive:
-        - 'null'
-        # TODO(dfox): binary alias?
-    negative:
-        - '{}'
-        - '[]'
+  # Can not accept this as a BODY argument due to NoOptionalBinaryBodyParamValidator
```

Comment:
I'm referring to this validator in the conjure compiler:
https://github.com/palantir/conjure/blob/d956c4a0692094647cc95331c691aa1c00d4b3b9/conjure-core/src/main/java/com/palantir/conjure/defs/validator/EndpointDefinitionValidator.java#L152